graylog2-server/src/main/java/org/graylog2/plugin/configuration/ConfigurationRequest.java

```diff
@@ -150,6 +152,8 @@ public Configuration filter(Configuration config) {
                     }
                     break;
                 case TextField.FIELD_TYPE:
+                case ListField.FIELD_TYPE:
```

Comment:
We also need to add a case for `ListField.FIELD_TYPE` in the `check()` method in this class.